deployments/clowder.yaml

```diff
@@ -154,6 +168,20 @@ objects:
                     name: app-secret
               - name: CLIENTS_RBAC_BASE_URL
                 value: "${CLIENTS_RBAC_BASE_URL}"
+              - name: CLIENTS_PENDO_BASE_URL
```

Comment:
A Q. The `db-tool jwk refresh` command is not planned to use the direct send to Pendo. I.e. the config is not needed here.  But that was the same with RBAC. Or was it needed?

Reply:
You are totally right! thanks! I will remove from the containers that does not need them!